fern/docs/pages/user-guide/benchmarking/kvbench/commands.md

```
@@ -0,0 +1,494 @@
+```bash
+python main.py plan \
+  --model ./examples/model_deepseek_r1.yaml \
+  --model_config ./examples/block-tp1-pp16.yaml \
+  --backend GDS \
+  --source gpu \
+  --etcd-endpoints "http://localhost:2379"
+```
+
+**Output:**
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)